app/views/shared/_claim.html.haml

```diff
@@ -4,58 +4,73 @@
 %h3.govuk-heading-m
   = t('.claim_type')
 
-= govuk_summary_list do
+= govuk_summary_list do |summary_list|
   - unless claim.providers_ref.blank?
-    = govuk_summary_list_row_collection('Provider reference number') do
-      = claim.providers_ref
+    - summary_list.with_row do |row|
+      - row.with_key(text: 'Provider reference number')
```

Comment:
I think this could come from a translation file (`en.shared.new_claim_and_case_type_details.provider_reference`) rather than be hardcoded but as it was like this before I don't think it's urgent that we change it now.